### PR TITLE
pypright: Remove ignored files that do not exist anymore

### PR DESCRIPTION
**Pull request by stefanberger**

### CODE REVIEW

**Review by THS-on — APPROVED** [review]
`test/test_ima_dm.py` still exists, but we can remove it from the ignore list.

**Comment by stefanberger** [comment]
> `test/test_ima_dm.py` still exists, but we can remove it from the ignore list.

Right, restored that one in the list.

**Comment by stefanberger** [comment]
@THS-on Iirc it was an class-upcast that was causing the pyright failures in test_ima_dm.py.

This here resolves it:
```
-            ima_buf: ast.ImaBuf = table_load_verity.mode
+            ima_buf: ast.ImaBuf = cast(ast.ImaBuf, table_load_verity.mode)
```

Wdyt?

**Comment by THS-on** [comment]
A cast fine for the test.

**Comment by stefanberger** [comment]
> A cast fine for the test.
Fixed up this test and added to this PR now.

File: pyproject.toml [hunk 1]
```diff
@@ -27,9 +27,6 @@ ignore = [
     "keylime/da/examples/redis.py",
     "keylime/da/examples/file.py",
     "keylime/da/record.py",
-    "test/oldtest.py",
-    "test/test_ima_dm.py",
-    "test/test_restful.py",
```
Comment (aplanas) [inline pyproject.toml:32]: Didn't we propose to ignore all the tests? IMHO those should not be type checked ...

**Review by aplanas — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (stefanberger) [reply pyproject.toml:32]: Now it's done and having the test cases type-checked as well caused me to fix a missing field in the Policies type, which does have some benefits already.

**Pull request merged**